magento/magento2-login-as-customer
# Removed "Login As Customer" action from Orders & Customers grid

### Description (*)
- Removed "Login As Customer" action from Orders grid in the admin panel
- Removed "Login As Customer" action from Customers grid in the admin panel

### Fixed Issues (if relevant)

1. magento/magento2-login-as-customer#15: Remove UI elements related to "Login as Customer" from Admin Panel grids

### Manual testing scenarios (*)
1. Login into Magento 2 admin panel. Go to Customer > All Customers. On the Customers grid you should not see  "Login As Customer" action or column "Login"
2. Login into Magento 2 admin panel. Go to Sales > Orders. On the Order grid you should not see  "Login As Customer" action or column "Action II"


### Contribution checklist (*)
 - [+] Pull request has a meaningful description of its purpose
 - [+] All commits are accompanied by meaningful commit messages
